fastapi-challenges/blog-api/devashmit/main.py

```
@@ -0,0 +1,63 @@
+Base.metadata.create_all(bind=engine)
+app = FastAPI(title="Blog API", version="1.0.0")
+
+class PostCreate(BaseModel): title: str; content: str; author: Optional[str]="Anonymous"
+class PostUpdate(BaseModel): title: Optional[str]=None; content: Optional[str]=None
+class PostOut(BaseModel):
+    id: int; title: str; content: str; author: str; created_at: str; updated_at: str
+    model_config={"from_attributes":True}
+    def model_post_init(self,_): self.created_at=str(self.created_at); self.updated_at=str(self.updated_at)
+class CommentCreate(BaseModel): body: str; author: Optional[str]="Anonymous"
+class CommentOut(BaseModel):
+    id: int; body: str; author: str; post_id: int; created_at: str
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)